packages/coinstac-ui/app/render/components/user/user-account-controller.jsx

```diff
@@ -25,6 +25,7 @@ class UserAccountController extends Component {
       });
     })
     .catch((err) => {
+      console.log(err);
```

Comment:
?

Reply:
Yeah, this can be taken out. Just a bit quicker for me to reference the console output rather than the log.